samples/nrf9160/coap_client/README.rst

```diff
@@ -32,13 +45,21 @@ Testing
 
 After programming the sample and all prerequisites to the board, test it by performing the following steps:
 
-1. Connect the USB cable and power on or reset your nRF9160 DK.
-#. Open a terminal emulator and observe that the kit prints the following information::
+1. Connect your nRF9160 DK to the PC using USB cable and power on or reset your nRF9160 DK.
+#. Open a terminal emulator and observe that the following information is displayed::
 
        The nRF CoAP client sample started
-#. Observe that the kit sends periodic CoAP GET requests after it gets LTE connection.
-   The kit sends requests to the configured server (``CONFIG_COAP_SERVER_HOSTNAME``) for a configured resource (``CONFIG_COAP_RESOURCE``).
-#. Observe that the kit either prints the reponse data received from the server or indicates a time-out.
+#. Observe that the following AT commands are displayed on the terminal emulator:
+	   
+	   * :command:`AT+COPS=3,2` which sets format of :command:`+COPS` read command response as numeric
+	   * :command:`AT+COPS?` which reads the current network selection mode and network operator name
+	   * :command:`AT%XCBAND`  which reads the current E-UTRA band
+	   * :command:`AT+CGDCONT?` which configures connection parameters
+	   * :command:`AT+CGACT?` which reads the PDP connections and states
+	
+#. Observe that the discovered IP address of the public CoAP server is displayed on the terminal emulator.
+#. Observe that the nRF9160 DK sends periodic CoAP GET requests to the configured server for a configured resource after it gets LTE connection.
+#. Observe that the sample either displays the response data received from the server or indicates a time-out on the terminal.
```

Comment:
Should we add more information on the response data? Are the CoAP codes (https://infocenter.nordicsemi.com/topic/sdk_nrf5_v16.0.0/group__iot__sdk__coap__codes.html#ga94f4c37162ba4eacbfe25f807b8dc4e6 ) relevant here?

Reply:
Well, CoAP codes are part of the CoAP specification, so yes these are relevant. But I think that if we decide to link some resource on them, we should link to the RFC (https://tools.ietf.org/html/rfc7252#section-12.1.2), not some other CoAP implementation docs.

Reply:
Added a link to RFC.